README.md

```diff
@@ -2,7 +2,7 @@
 
 This project aims to bring Google's new [Material Design](https://www.google.com/design/spec/material-design/introduction.html) to Ember. The goal is to encapsulate everything possible in Ember components. This project is packaged as an [Ember-cli](http://www.ember-cli.com/) addon.
 
-[Explore Ember Paper docs »](https://miguelcobain.github.io/ember-paper/)
+[Explore Ember Paper docs »](https://adopted-ember-addons.github.io/ember-paper/)
```

Comment:
this is now actually https://ember-paper.netlify.app so that we can get deploy previews on PRs 💪